tasks/main.yml

```diff
@@ -1,6 +1,6 @@
 ---
-- include_tasks: parse-version.yml
-  when: datadog_agent_version | length > 0
+
+- include_tasks: set-parse-version.yml
   run_once: true
```

Comment:
What is I set different version per host ? I think using `run_once` here would introduce a buggy behavior, no ?